stable/artifactory-ha/values.yaml

```diff
@@ -1064,6 +1066,18 @@ nginx:
       if ($http_x_forwarded_proto = '') {
         set $http_x_forwarded_proto  $scheme;
       }
+      {{- if .Values.nginx.service.ssloffload }}
+      if ($host = $server_addr) {
```

Comment:
In what case would `$host` be different from the `$server_addr` ? From looking at the nginx docs, seems like they are the same. But maybe I'm missing something..?

Reply:
@danielezer Removed the Force SSL redirect part.